cypress/e2e/users_spec/user_avatar.cy.ts

```diff
@@ -0,0 +1,22 @@
+import { UserAvatar } from "@/pageObject/Users/UserAvatar";
+
+describe("User Profile Avatar Modification", () => {
+  const userAvatar = new UserAvatar("dev-nurse");
+  beforeEach(() => {
+    cy.loginByApi("devnurse");
```

Comment:
```suggestion
    cy.loginByApi("devnurse");
```

create a new user and add it to the fixture since we are connected to the develop backend